### PR TITLE
Postman instruction clarification for readme.md

### PR DESCRIPTION
**Pull request by watchtheblur**
### Background/Context
Postman Instructions needs additional information

### Before and After
Dev environment needs to have the right addresses in initial and current value fields

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,21 +77,21 @@ You can access the database through the Adminer front-end or using a local Postg
 5. Create a superuser so that you can log into `http://localhost:8000/admin` by running the following in your terminal:
 
 ```bash
-$ docker-compose run --rm app ./manage.py createsuperuser
+$ docker-compose run --rm app python ./manage.py createsuperuser
```
Comment (lpatmo) [inline README.md:80]: Whoa, I thought these `python` changes were merged in already from https://github.com/codebuddies/backend/pull/147. Wonder why they appeared in your branch again.

**Comment by lpatmo** [comment]
Saving this screenshot to use for the image url:

Screen Shot 2020-07-28 at 7 53 03 PM

File: README.md [hunk 2]
```diff
@@ -193,16 +193,16 @@ Linux,
...
-4. Name your environment `dev` and create a variable called `api_url`. Give it a value of `https://localhost:8000`, which is the URL of your Django dev environment when it is running.
-5. Now, as long you have the Django app (https://localhost:8000) running, you should be able to make requests like POST Create User and POST Authenticate.
+4. Name your environment `dev` and create a variable called `api_url`.  For both the "Initial Value" and "Current Value," fill in `http://localhost:8000` which is the URL of your Django dev environment when it is running.
+5. Now, as long you have the Django app (http://localhost:8000) running, you should be able to make requests like POST Create User and POST Authenticate.
    Click on this link to see what you should expect: https://imgur.com/hd9VB6k
 
 - `POST` Create User will create a new user in your `localhost:8000` running Django app,
 - making a request to `POST Authenticate` will authenticate whether or not that user exists.
 
 ![screenshot of Postman environment variable setup](https://i.imgur.com/6Uq9XQp.png)
```
Comment (lpatmo) [inline README.md:203]: Let's also update this screenshot of the Postman environment variable setup to be this image:

https://user-images.githubusercontent.com/4512699/88751261-14fa9280-d10c-11ea-80c4-2934c27254ad.png

Comment (watchtheblur) [reply README.md]: Update screenshot for Postman configuration

File: README.md (hunk 1 above)
Comment (watchtheblur) [reply README.md]: Solved by rebasing

**Pull request merged**